configs/14.0/deb/monolithic/rules

```diff
@@ -38,7 +38,7 @@ ifeq (__USE_SYSTEMD__, yes)
 endif
 
 # Copy the packages' files.
-	dh_install --fail-missing
+	dh_missing --fail-missing
```

Comment:
same comment as above, of course.

Reply:
@ThinkOpenly you're right. I updated the patch.